web/src/components/Cards/jobShowCarousel.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import  {Carousel,Card} from 'react-bootstrap'
+
+const items = (source) => {
```

Comment:
Is this a component? If so, two notes: 1) the name of a component should be capitalized, and 2) the only parameter should be `props`:
```js
const Items = ({ source }) => {
```